api/config/v1alpha1/envoyproxy_types.go

```diff
@@ -44,6 +44,10 @@ type EnvoyProxySpec struct {
 	// +kubebuilder:default={level: {system: info}}
 	Logging ProxyLogging `json:"logging,omitempty"`
 
+	// AccessLoggings defines access logging parameters for managed proxies.
```

Comment:
should this be put under a top level `Observability` field ?

Reply:
or `Telemetry` ?

Reply:
this comment can be ignored, wanted to bring this up to have a discussion, in most projects `logging`, `tracing` and `metrics` are not clubbed under a top level field, but doing so makes it easier for the user to remember to enable all relevant telemetry signals